### ISSUE TITLE
[1.1.0] - import mzml2isa - ValueError: could not find `owl:Ontology` element

### ISSUE THREAD

**Issue by DrYak** [ISA-tools/mzml2isa#49]
The [`import mzml2isa` test of the bioconda package](https://github.com/bioconda/bioconda-recipes/blob/740665bbbbb16b9c66eb9c10b450ad757e3f9b25/recipes/mzml2isa/meta.yaml#L31) for the auto bumped version 1.1.0 is currently [failing with an error message](https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=22723&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=7df82132-b284-504b-53d6-7d3e63519572&l=799) :

```console
09:16:47 BIOCONDA INFO (OUT)   File "/opt/conda/conda-bld/mzml2isa_1665911628804/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.9/site-packages/pronto/parsers/rdfxml.py", line 89, in parse_from
09:16:47 BIOCONDA INFO (OUT)     raise ValueError("could not find `owl:Ontology` element")
09:16:47 BIOCONDA INFO (OUT) ValueError: could not find `owl:Ontology` element
```

Is there something wrong?

**Comment by DrYak** [comment]
Culprit seems to be [here](https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=22723&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=7df82132-b284-504b-53d6-7d3e63519572&l=775):
```text
09:16:47 BIOCONDA INFO (OUT)   File "/opt/conda/conda-bld/mzml2isa_1665911628804/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.9/site-packages/mzml2isa/mzml.py", line 64, in <module>
09:16:47 BIOCONDA INFO (OUT)     class MzMLFile(object):
09:16:47 BIOCONDA INFO (OUT)   File "/opt/conda/conda-bld/mzml2isa_1665911628804/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.9/site-packages/mzml2isa/mzml.py", line 105, in MzMLFile
09:16:47 BIOCONDA INFO (OUT)     _VOCABULARY = pronto.Ontology(filename)
```

(BTW, same error message as [that pronto issue](https://github.com/althonos/pronto/issues/68))

**Comment by althonos** [comment]
This issue is raised when `pronto` fails to load an XML document, which happens typically when trying to resolve a missing ontology on `http://purl.obolibrary.org`, which may return an XML listing (that's a bug in itself). I just noticed that `stato.owl` was not embedded as expected in the `mzml2isa` distribution, so it's likely that `pronto` was trying to find it online and failed. I'll make a new release fix the missing file included.

**Comment by althonos** [comment]
Looks like the issue is fixed in `v1.1.1`, the Bioconda build is fine. :+1:
